src/main/java/com/liferay/ide/idea/ui/actions/AbstractLiferayGradleTaskAction.java

```diff
@@ -169,6 +173,23 @@ public void processTerminated(
 				}
 
 			});
+
+		messageBusConnection.subscribe(
+			RunContentManager.TOPIC,
+			new RunContentWithExecutorListener() {
+
+				@Override
+				public void contentRemoved(@Nullable RunContentDescriptor descriptor, @NotNull Executor executor) {
+					ProcessHandler processHandler = descriptor.getProcessHandler();
+
+					processHandler.destroyProcess();
```

Comment:
@jtydhr88 

Is this safe to execute for any contentRemoved?  Are we sure that this will only be called for our processes that we started by this action?